spaceship/spec/connect_api/models/review_submission_item_spec.rb

```diff
@@ -0,0 +1,45 @@
+describe Spaceship::ConnectAPI::ReviewSubmissionItem do
+  let(:mock_tunes_client) { double('tunes_client') }
+  let(:username) { 'spaceship@krausefx.com' }
+  let(:password) { 'so_secret' }
+
+  before do
+    allow(mock_tunes_client).to receive(:team_id).and_return("123")
+    allow(mock_tunes_client).to receive(:select_team)
+    allow(mock_tunes_client).to receive(:csrf_tokens)
+    allow(Spaceship::TunesClient).to receive(:login).and_return(mock_tunes_client)
+    Spaceship::ConnectAPI.login(username, password, use_portal: false, use_tunes: true)
+  end
```

Comment:
A lot of test files work with just before { Spaceship::Tunes.login } in order to initialise the client used with Spaceship::ConnectAPI, however I could not make it work here and I don't know why. 🙁

I saw other tests files (like /models/app_spec.rb) that use this instead and it works for me here. If you think it is worth investigating why approach 1 does not work (or if someone asks you in the upstream PR), we can try to investigate further.

Reply:
I have no idea, to be honest, but if this works and it's used throughout the codebase I think it's fine, we're not inventing anything strange 👍